### ISSUE TITLE
Model method to normalize input values

### ISSUE THREAD

**Issue by avit** [dimko/normalizr#3]
I'd like to have a class method such as `normalize_values` that does something like this:

```
input = {product_code: "  ABC123  "}
clean_input = User.normalize_values(input)
#=> {product_code: "ABC123"}
User.where(clean_input)
```

This would effectively build a new model instance to push values through the normalizer, and read them back. Is that within scope of your gem for a pull request?

**Comment by dimko** [comment]
Yeah, it can be useful. But I guess it will be quite complicated to implement.
